Documentation/concepts/scalability/identity-relevant-labels.rst

```diff
@@ -0,0 +1,111 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _identity-relevant-labels:
+
+*********************************
+Limiting Identity-Relevant Labels
+*********************************
+
+We recommend that operators with larger environments limit the set of
+identity-relevant labels to avoid frequent creation of new security identities.
+Many Kubernetes labels are not useful for policy enforcement or visibility. A
+few good examples of such labels include timestamps or hashes. These labels,
+when included in evaluation, cause Cilium to generate a unique identity for each
+pod instead of a single identity for all of the pods that comprise a service or
+application.
+
+By default, Cilium evaluates the following labels:
+
+=================================== ==================================================
+Label                               Description
+----------------------------------- --------------------------------------------------
+``k8s:io.kubernetes.pod.namespace`` Include all ``io.kubernetes.pod.namespace`` labels
+``k8s:app.kubernetes.io``           Include all ``app.kubernetes.io`` labels
+``k8s:!io.kubernetes``              Ignore all ``io.kubernetes`` labels
+``k8s:kubernetes.io``               Ignore all other ``kubernetes.io`` labels
```

Comment:
I don't fully understand the backport process, so just ask my question here. 

As this file is added, do you want to backport this PR as well https://github.com/cilium/cilium/pull/12639 ? And normally, how do we decide which PR should be backported ? Thanks

Reply:
Broadly, anyone can nominate PRs for backport based the criteria described [in the docs](https://docs.cilium.io/en/v1.8/contributing/release/backports/). During review of the PR, usually someone will add the labels and reviewers then have the opportunity to see whether the PR is appropriate for backport & discuss during review. Typically for docs changes this is fairly loose, we want the docs to be as clear as possible on all relevant branches.

Given that you've raised this and it seems like the dependent PR is already backported, I'd say it's reasonable to also nominate #12639 for backport. Thanks for bringing it up :+1:

Reply:
oh thanks, I might have missed the docs :+1:

Reply:
Added #12639 @sayboras. Thanks for bringing this up!